catalog/couchdb/couchdb-cluster.bom

```diff
@@ -34,32 +34,54 @@ brooklyn.catalog:
     item:
       services:
       - type: org.apache.brooklyn.entity.nosql.couchdb.CouchDBNode
+        id: couchdb-cluster-node
+        provisioning.properties:
+          inboundPorts:
+          - 22
+          - 5986
+          - 5984
+          - 4369
```

Comment:
This worries me
![screen shot 2017-06-08 at 17 55 56](https://user-images.githubusercontent.com/207962/26940747-c8f939e2-4c73-11e7-90ad-a23fa71678b7.png)